### DISCUSSION TITLE
Can't find ID of working Container

### DISCUSSION THREAD

**Discussion by hollysheep** [containers/podman#24683] [General]
I'm know sure whether this is a Bug or something else. This https://github.com/netbrain/zwift is the first and only Project i use with Docker (Podman). First i installed Podman on Ubuntu 24.04, then run the script from the Projects git and started it with the command "zwift". Then it's downloading 4GB of data. All is working very good. But i can't create a backup because there is not ID. When i run `podman ps -a` i only see the headers:

```
podman ps -a
CONTAINER ID  IMAGE       COMMAND     CREATED     STATUS      PORTS       NAMES
```

Is that normal that a working Docker Container has no ID?

**Comment by baude** [comment]
that means there are no containers running for that user.  it would be more helpful if you filled out the issue template with podman info and the required information.  else, it is hard to speculate what is going on.  Is it possible you ran zwift as one user and podman as another?

**Comment by hollysheep** [comment]
Please tell me what Information you need?

Normally i start Zwift using the Command `zwift` in a Terminal. So i don't start it with an other user. There is no other Linux User on my System. The complete command is:

`podman run --rm --privileged --network bridge --name zwift-ich --security-opt label=disable --hostname ich-zwift -e DISPLAY=:0.0 -e ZWIFT_UID=1000 -e ZWIFT_GID=1000 -e PULSE_SERVER=/run/user/1000/pulse/native -v zwift-ich:/home/user/.wine/drive_c/users/user/Documents/Zwift -v /run/user/1000/pulse:/run/user/1000/pulse --device=/dev/dri:/dev/dri -e DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/1000/bus -v /run/user/1000/bus:/run/user/1000/bus -e XAUTHORITY=/home/ich/.Xauthority -v /tmp/.X11-unix:/tmp/.X11-unix -v /home/ich/.Xauthority:/home/ich/.Xauthority --userns keep-id:uid=1000,gid=1000 docker.io/netbrain/zwift:latest`

Here you can see the Container has a Name (zwift-$USER). Is this the 'user' you mean?

**Comment by hollysheep** [comment]
I tried to backup the folder 'containers' but that failed because of missing permissions. I found out that some of the folders have different owners. In `~/.local/share/containers/storage/overlay` i have a huge number of folders. One of them has an other owner:

```
drwx------ 5 ich    ich    4.0K Nov 12 22:41 136f85040501f1115be4abe2399600251f102ab1aabe44636ea79172c4895a58
drwx------ 5 ich    ich    4.0K Nov 12 22:41 150d14e86cd00b0ce8d5111422c85096b1db45d646a57a37c9608afb39c288be
drwx------ 5 ich    ich    4.0K Nov 12 22:41 27c4301d78e983995b45053c01f3c97f7db4881850fb576ee14b0fd869771529
drwx------ 5 ich    ich    4.0K Nov 12 22:41 2ed88c7bbe393f95f8519f4c9b8d0ca171831a77c9d4405f315a8aab1ad7c676
drwx------ 5 ich    ich    4.0K Nov 12 22:41 2f45561c24a23d89fbb6382288ffe5d735f00155190100c53293dcc8e6205808
drwx------ 5 ich    ich    4.0K Nov 12 21:45 3d62b55e0682f861052b2a400406887e8831c35b7737b0aae65376398355e1ed
drwx------ 5 ich    ich    4.0K Nov 12 21:45 4b1de5172853d4c959c0d44794e7e4f129b9a51ababbbbd7c3a662bec82bd5e4
drwx------ 5 ich    ich    4.0K Nov 12 22:41 5b020d1441706c6cf54ae5aedf4f0240805b32ad3a5fbfb882947ddb189949bc
drwx------ 5 100000 100000 4.0K Nov 13 01:25 5b5b0f8adbd05220c12c782d8f461fb2bda00fd2b548fd95fa71b76b7d2cda12
drwx------ 5 ich    ich    4.0K Nov 13 01:23 5e66a947e7f28f0df388f526e615e2e5fe252ad5aab45a1b129f4e69b030028e
drwx------ 5 ich    ich    4.0K Nov 12 22:41 6aa60d3a096d42897e60b8e28dfe7370f812e6e97c0ab6ed3575792e2030c9d3
drwx------ 5 ich    ich    4.0K Nov 12 22:41 773a99d528fe6dad58402b310aa2bfce7ae144a00a90784f8205d370748d22ca
drwx------ 5 ich    ich    4.0K Nov 12 20:40 7e302cc516e5a310f8ac10cf8111741bae8abe15fad72259aec79f128768fe7a
drwx------ 5 ich    ich    4.0K Nov 13 01:22 84e2cb94a777f90b6935419332816d8c66a0c4d297aac817182473144483c51f
drwx------ 5 ich    ich    4.0K Nov 12 21:45 96740c9cdb7d0018ff3f5046cef323767ffe0b7aed9026914a5ed72bb4677aea
drwx------ 5 ich    ich    4.0K Nov 12 22:41 9997242060a9cbefa8f121968b61d8edb95c9049b2bb4a54f65df37e96e65f28
drwx------ 5 ich    ich    4.0K Nov 12 22:41 9c412a0816ef22a604f18d8f9d0b30c879be81e91cd5661a879892ea29f6d1d0
drwx------ 5 ich    ich    4.0K Nov 12 22:41 a24d2ddca62e57cbdbc4b46899daf38451cd958b6092e4123232029dee4f3af8
drwx------ 5 ich    ich    4.0K Nov 12 22:41 a926f1aa71bb0a8a578994ecc88ec80f6776eb9552ea2a8c46d039998d6b9271
drwx------ 5 ich    ich    4.0K Nov 12 21:45 b7146bc3b5878edefe01c1225fa51525e25c9d8924585f68608bdf6950b59cd1
drwx------ 5 ich    ich    4.0K Nov 12 22:41 ba3c96459edae87cceff59f3b3a90d9338db02fe9b70c52e2b0a55284b0dc810
drwx------ 5 ich    ich    4.0K Nov 12 20:45 d1756e6209c6b0ea1b62b76e68de181de8615eeb7ca89995add50740d2f65bcd
drwx------ 5 ich    ich    4.0K Nov 12 22:41 d7fd64894da17b1a6f74653d70c1f45b70a075c4943b7783948421c0eb0355ff
drwx------ 5 ich    ich    4.0K Nov 13 01:23 e92d496dc80a6e3b7e28b8cd2199f1320ef8502bd9d2e01e4afeb8284f7c3322
drwx------ 5 ich    ich    4.0K Nov 12 22:41 ec64ca24ede9b84032d91e322c99e5ef5a9b0d58ed147ea3dc37b773e051cb59
drwx------ 5 ich    ich    4.0K Nov 12 22:41 eedbd9ae6149a328d9c282bf9d0e6bc485e3734758ca8a25c089a86972f226e0
drwx------ 6 ich    ich    4.0K Nov 12 20:40 fd37bdd9c2276efc14f6857abb3c47d91509a784fb0fd00a7fc410117b1302cd
```

Some other sub folders have the same problem:

in `containers/storage/overlay/d1756e6209c6b0ea1b62b76e68de181de8615eeb7ca89995add50740d2f65bcd/diff/var/lib/apt/lists/partial`

```
drwxr-xr-x 2 100041 ich 4.0K Oct 25 10:02 auxfiles
drwx------ 2 100041 ich 4.0K Oct 25 10:02 partial
```

**Comment by Luap99** [comment]
> podman run --rm

if you run with --rm the container will be removed when it exists so that is normal when your container exists right away

Also permissions are expected, file uids/gids can be naything within the user namespace range configured in /etc/subuid /etc/subgid.
Use `podman unshare` to access those files if needed
